code/modules/culture_descriptor/culture/citizenship_hidden.dm

```diff
@@ -1,20 +1,20 @@
-/decl/cultural_info/culture/hidden
+/decl/cultural_info/citizenship/hidden
```

Comment:
Hard no on repathing this stuff. This will be a _nightmare_ for pulling in changes from upstream.

Adding a new citizenship class as a many-to-many mapping of specific locations and cultures is a good idea - we should put our new cultures in `mods/`, not modify the base code.